src/vs/base/browser/ui/selectBox/selectBoxCustom.ts

```diff
@@ -848,10 +848,14 @@ export class SelectBoxList implements ISelectBoxDelegate, IListVirtualDelegate<I
 
 		this.selectionDetailsPane.innerText = '';
 		const selectedIndex = e.indexes[0];
-		const description = this.options[selectedIndex].description;
-		const descriptionIsMarkdown = this.options[selectedIndex].descriptionIsMarkdown;
+		if (!selectedIndex || !this.options[selectedIndex]) {
```

Comment:
@cleidigh shouldn't that check for `typeof selectedIndex !== 'number'` instead of checking like this, which would mean if `selectedIndex === 0` it would also match that condition.